ports/espressif/mpconfigport.mk

```diff
@@ -2,7 +2,7 @@
 INTERNAL_FLASH_FILESYSTEM = 1
 
 # Internal math library is substantially smaller than toolchain one
-INTERNAL_LIBM = 1
+INTERNAL_LIBM = 0
```

Comment:
Why did you change this to use the ESP-IDF libm?

Reply:
There is a rom blob (can't remember which) that links against `floor()`. Our implementation has it for doubles but it didn't compile. So, I switched it assuming that's what espressif uses.